workloadDriver/src/main/java/com/vmware/weathervane/workloadDriver/common/core/loadPath/FindMaxLoadPath.java

```diff
@@ -131,7 +131,7 @@ public void initialize(String runName, String workloadName, Workload workload, L
 		logger.debug("initialize " + this.getName() + ": minUsers = {}, maxUsers = {}", getMinUsers(), maxUsers);
 
 		this.maxUsers = workload.getMaxUsers();
-		this.initialRampRateStep = maxUsers / 20;
+		this.initialRampRateStep = maxUsers / 40;
```

Comment:
Does this now set the initialRampRateStep to 750 on small?

Reply:
Fixed by adding initialRampRateStep parameter